tf2_eigen/include/tf2_eigen/tf2_eigen.hpp

```diff
@@ -148,7 +148,9 @@ void doTransform(
   const geometry_msgs::msg::TransformStamped & transform)
 {
   // References:
-  // https://core.ac.uk/download/pdf/154240607.pdf, https://www.seas.upenn.edu/~meam520/notes02/Forces8.pdf
+  // https://core.ac.uk/download/pdf/154240607.pdf
+  // https://www.seas.upenn.edu/~meam520/notes02/Forces8.pdf
```

Comment:
This URL is now dead, so I think we can get rid of it

Reply:
Its available on archive. 
But the things in that reference is more confusing. The modern robotics book is better.

[Forces8.pdf](https://github.com/ros2/geometry2/files/12370578/Forces8.pdf)
Here is the pdf if you did not already get it.